change/@ni-nimble-components-9da2496e-e544-4019-a666-d7fca3046771.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You should mark #2022 as being fixed by this PR. You mention it in your PR description, but it isn't marked as being fixed.

Reply:
I think I did that. Once this PR is submitted, I think that issue will be marked as closed.